x-pack/plugins/upgrade_assistant/__jest__/client_integration/overview/fix_logs_step/fix_logs_step.test.tsx

```diff
@@ -33,14 +33,49 @@ describe('Overview - Fix deprecation logs step', () => {
     server.restore();
   });
 
+  describe('Step status', () => {
```

Comment:
@sabarasaba I added some logic that will set the step state to incomplete if the API poll receives `data` followed by an `error`. There's similar logic for the "Back up data" step, and I added a [test](https://github.com/elastic/kibana/pull/111243/files#diff-84d8cce1aad966faf177198b8104065830ec25a4d10f51fc9fbb7951b28f2e8fR160) to cover it. Could you add a similar test here in a followup PR?

Reply:
Will do after we merge this one in 🔧

Reply:
Added with https://github.com/elastic/kibana/pull/111701